pulp_ansible/app/serializers.py

```diff
@@ -83,6 +84,26 @@ class Meta:
         model = RoleRemote
 
 
+class AnsibleGitRemoteSerializer(RemoteSerializer):
+    """
+    A serializer for Git Collection Remotes.
+    """
+
+    metadata_only = serializers.BooleanField(
+        help_text="If True, only metadata about the content will be stored in Pulp. Clients will retrieve content from the remote URL.",
+        required=False,
+    )
+
+    class Meta:
+        fields_list = []
+        for field in RemoteSerializer.Meta.fields:
+            if field != "policy":
+                fields_list.append(field)
+        fields_list.append("metadata_only")
+        fields = tuple(fields_list)
```

Comment:
Instead of this just use the `excludes` variable to remove `policy`: https://www.django-rest-framework.org/api-guide/serializers/#specifying-which-fields-to-include

Reply:
How about:

```python
tuple(set(RemoteSerializer.Meta.fields) - {'policy'}) + ('metadata_only',)
```